lib/tasks/dump.rake

```diff
@@ -47,7 +47,13 @@ namespace :db do
       filename = File.join('tmp', "#{Time.now.strftime('%Y%m%d%H%M%S')}_dump.psql")
 
       shell_working "exporting unanonymised database data to #{filename}..." do
-        cmd = "pg_dump $DATABASE_URL --no-owner --no-privileges --no-password #{sensitive_table_exclusions} #{unneeded_table_exclusions} -f #{filename}"
+        if args.local.eql?('local')
+          with_config do |host, db, user|
+            cmd = "pg_dump --no-owner --no-privileges --no-password #{sensitive_table_exclusions} #{unneeded_table_exclusions} #{db} -f #{filename}"
+          end
```

Comment:
`host` and `user` appears to be unused. Are they missing from the command or can they be removed (on lines 271 and 273)?

Reply:
yeah, i mean to tidy this up. it was from previous attempts.

Reply:
amended